articles/uploading-images-to-cloudinary-from-django-application/index.md

```diff
@@ -0,0 +1,270 @@
+
+```python
+INSTALLED_APPS = [
+
+    'django.contrib.admin',
+    
+    'django.contrib.auth',
+    
+    'django.contrib.contenttypes',
+    
+    'django.contrib.sessions',
+    
+    'django.contrib.messages',
+    
+    'django.contrib.staticfiles',
+    
+    'photos',
+    ]
+```
+
+### Setting up Cloudinary
+
+Now lets head over to [cloudinary](https://cloudinary.com/) website to create an account.
+Click on the Sign Up button and fill in your details to create an account.
+
+### Installation
+
+Install the cloudinary module in our project
+
+```bash
+$ pip install cloudinary
+
+```
+
+Next thing is to add Cloudinary to the list of installed apps in settings.py
+
+
+```python
+INSTALLED_APPS = [
+
+    'django.contrib.admin',
+    
+    'django.contrib.auth',
+    
+    'django.contrib.contenttypes',
+    
+    'django.contrib.sessions',
+    
+    'django.contrib.messages',
+    
+    'django.contrib.staticfiles',
+    
+    'photos',
+    'cloudinary'
+    ]
+```
+
+We will also need to include Cloudinary's python classes in settings.py
+
+```python
+    import cloudinary
+    import cloudinary.uploader
+    import cloudinary.api
+```
+
+### Configuration
+
+To use the Cloudinary Django library, we have to configure at least our cloud_name, api_key and api_secret .
+we can find our account-specific configuration credentials in the Dashboard page of the account console.
+
+![Cloudinary Dashboard](/engineering-education/uploading-images-to-cloudinary-from-django-application/dash.png)
+
+```python
+cloudinary.config( 
+  cloud_name = "your_cloud_name", 
+  api_key = "Your_api_key", 
+  api_secret = "Your_api_secret" 
+)
+```
+
+### Creating a model
+
+In the “photos” directory, edit the models.py file and add the following lines of code to it.
+
+```python
+    from django.db import models
+    from cloudinary.models import CloudinaryField
+
+    class photos(models.Model):
+        Tittle = models.CharField(max_length=100)
+        image = CloudinaryField('image')
+```
+
+Now let's migrate our model to the database by running the commands bellow
+
+```bash
+# migrating the app and database changes
+$ python manage.py makemigrations
+
+# final migrations
+$ python manage.py migrate
+```
+
+Then we create a superuser by running the command below
+
+```bash
+ $ python manage.py createsuperuser 
+ 
+ ```
+
+
+Now let's register the photos model in the admin.py file so we can modify it in the Django admin section.
+
+```python
+from django.contrib import admin
+from .models import photos
+
+admin.site.register(photos)
+
+```
+
+Let's now login to the admin page.
+
+To login to the admin section, go to this link [localhost:8000/admin](localhost:800/admin) and login with our just created superuser details.
+
+![Django Admin](/engineering-education/uploading-images-to-cloudinary-from-django-application/admin.png)
+
+![](/engineering-education/uploading-images-to-cloudinary-from-django-application/django-admin.png)
```

Comment:
It is always advised to add an image alternative.

And, add image source and footer as mentioned previously.